crypt.go

```diff
@@ -725,6 +740,24 @@ func InitializeLUKS2Container(devicePath, label string, key []byte, options *Ini
 				"--luks2-keyslots-size", fmt.Sprintf("%dk", options.KeyslotsAreaKiBSize))
 		}
 	}
+	if pbkdfOptions != nil {
```

Comment:
this logic here and repeated in the other functions could be  extracted as a method on *PbkdfLUKS2ContainerOptions

```
func (o *PbkdfLUKS2ContainerOptions) appendToArgs(args []string, fallbackIterTimeMs int) []string {
   ...
}
```